tests/e2e/scarb/simple_deps.rs

```diff
@@ -0,0 +1,162 @@
+use cairo_lang_test_utils::parse_test_file::TestRunnerResult;
+use cairo_lang_utils::ordered_hash_map::OrderedHashMap;
+use cairo_language_server::lsp;
+use indoc::indoc;
+use lsp_types::NumberOrString;
+
+use crate::support::normalize::normalize;
+use crate::support::sandbox;
+
+cairo_lang_test_utils::test_file_test!(
+    simple_deps,
+    "tests/test_data/scarb",
+    {
+        simple_deps: "simple_deps.txt"
+    },
+    test_simple_deps
+);
+
+fn test_simple_deps(
+    inputs: &OrderedHashMap<String, String>,
+    _args: &OrderedHashMap<String, String>,
+) -> TestRunnerResult {
+    let mut ls = sandbox! {
+        files {
+            "a/Scarb.toml" => inputs["a/Scarb.toml"].clone(),
+            "a/src/lib.cairo" => inputs["a/src/lib.cairo"].clone(),
+            "b/Scarb.toml" => inputs["b/Scarb.toml"].clone(),
+            "b/src/lib.cairo" => inputs["b/src/lib.cairo"].clone(),
+        }
+    };
+
+    assert!(ls.open_and_wait_for_diagnostics("a/src/lib.cairo").diagnostics.is_empty());
+
+    // Check if opening a triggers calculating diagnostics for b.
+    let diagnostics_from_b = ls.wait_for_diagnostics("b/src/lib.cairo");
+    assert_eq!(diagnostics_from_b.diagnostics.len(), 1);
+    assert_eq!(
+        diagnostics_from_b.diagnostics[0].code,
+        Some(NumberOrString::String("E0005".to_string()))
+    );
+
+    let analyzed_crates = ls.send_request::<lsp::ext::ViewAnalyzedCrates>(());
+
+    pretty_assertions::assert_eq!(normalize(&ls, analyzed_crates), indoc! {r#"
```

Comment:
if you're using fixtures, then maybe put this as an output and let the fixtures framework do the compasiron?

Reply:
my test haven't been using cairo-lang-test-utils, yours do